src/lib/routes/backoffice/BackOffice.js

```diff
@@ -2,13 +2,15 @@ import { Notifications } from '@components/Notifications';
 import { Sidebar } from '@components/backoffice/Sidebar';
 import React, { Component } from 'react';
 import BackOfficeRoutesSwitch from './BackOfficeRoutesSwitch';
+import PropTypes from 'prop-types';
 
 export class BackOffice extends Component {
   render() {
+    const { location } = this.props;
     return (
       <div className="backoffice">
         <div className="bo-sidebar">
-          <Sidebar />
+          <Sidebar location={location} />
```

Comment:
What is this fixing?

Reply:
The location here:
 https://github.com/inveniosoftware/react-invenio-app-ils/blob/e8c52f1f2c14a8461ca6fc67734ef498bdbd3cc1/src/lib/components/backoffice/Sidebar/Sidebar.js#L21 was empty before because the prop wasn't passed from the `BackOffice.js` and the links in the sidebar were not displayed as active.

Reply:
Nice catch!!!